common/time-service/src/mock.rs

```diff
@@ -179,7 +188,12 @@ impl MockTimeService {
 }
 
 impl TimeServiceTrait for MockTimeService {
-    fn now(&self) -> Duration {
+    fn now(&self) -> Instant {
+        let this = self.lock();
+        this.base_time + this.now
+    }
```

Comment:
Is there a reason `base_time` can't just be some constant arbitrary value?  Like 0?

Reply:
`Instant` is an opaque type; the only way to construct one is through `Instant::now`.